shae-riches/week7/mon/js/main.js

```diff
@@ -20,22 +20,19 @@ const queryMovieDatabase = function( movieTitle ) {
         $(`#results`)
             .empty()    //                  remove previous results    
             .prepend(`
-                <h2>'${ '#movieSearchInput' }' Search Results</h3>
-                ${ for (let i = 0; i < array.length; i++) {
-                    const element = array[i];
-                    
-                    }
+                <h2>'${ `${movieSearchInput}` }' Search Results</h3>
```

Comment:
You have passed in the value of `movieSearchInput` down below and use movieTitle in fetching the api. You should use movieTitle here for the search results